ml-service/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM python:3.9.7
+
+COPY train.py /app/
+RUN pip install numpy scikit-learn
```

Comment:
please add versions and put deps to requirements.txt 

reference: https://learnpython.com/blog/python-requirements-file/